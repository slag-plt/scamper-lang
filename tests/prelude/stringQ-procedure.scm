(string? (cons #t #f))
(string? 1)
(string? "bye")
(string? "a")
(string? 100)
(procedure? (cons #t #f))
(procedure? list)
(procedure? "bye")
(procedure? +)
(procedure? string-length)