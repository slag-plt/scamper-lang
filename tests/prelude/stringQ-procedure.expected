#f
#f
#t
#t
#f
#f
#t
#f
#t
#t
